### PR TITLE
Upgrade to eleventy v1.0.0 🎉 

### PR DESCRIPTION
**Pull request by mxmason**
## Summary
- Upgrades eleventy to `1.0.0`
- Upgrades minor versions of other packages
- Makes some code tweaks to complete migration to eleventy v1.0
- Sets `--incremental` during `eleventy-watch` and dogfoods npm scripts during `npm start`

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ const eleventyNavigationPlugin = require("@11ty/eleventy-navigation");
 const pluginSyntaxHighlight = require("@11ty/eleventy-plugin-syntaxhighlight");
 const eleventyPluginTOC = require("eleventy-plugin-nesting-toc");
 
-const slugify = require("slugify");
```
Comment (mxmason) [inline .eleventy.js:6]: 1️⃣ Eleventy uses `slugify` by default now

File: .eleventy.js [hunk 2]
```diff
@@ -62,20 +60,11 @@ module.exports = function (eleventyConfig) {
...
 
-	// Universal slug filter strips unsafe chars from URLs
-	eleventyConfig.addFilter("slugify", function (str) {
-		return slugify(str.replace(/<\/?("[^"]*"|'[^']*'|[^>])*(>|$)/g, ""), {
-			lower: true,
-			replacement: "-",
-			remove: /[*+~.·,()'"`´%!?¿:@»]/g,
-		});
-	});
-
```
Comment (mxmason) [inline .eleventy.js:78]: 2️⃣  deleted as part of 1️⃣

File: .eleventy.js [hunk 3]
```diff
@@ -15,7 +14,6 @@ module.exports = function (eleventyConfig) {
 		tags: ["h2", "h3"],
 		wrapperClass: "l-toc",
 	});
-	eleventyConfig.setDataDeepMerge(true);
```
Comment (mxmason) [inline .eleventy.js:18]: 3️⃣ eleventy sets this by default now

File: src/announcements/lets-support-inert.md [hunk 4]
```diff
@@ -69,7 +69,7 @@ We understand that money is tight for many communities and companies right now.
 Thank you for considering this initiative, and for your generosity.
 
 <p class="c-contribute__cta">
-	Contact us to <a href="{{ /contact/ | url }}">tell us about your pledge</a>.
+	Contact us to <a href="{{ '/contact/' | url }}">tell us about your pledge</a>.
```
Comment (mxmason) [inline src/announcements/lets-support-inert.md:72]: 4️⃣ eleventy got a little stricter about a this being a string, so I fixed it

File: .eleventy.js [hunk 5]
```diff
@@ -62,20 +60,11 @@ module.exports = function (eleventyConfig) {
 
 	// only content in the `posts/` directory
 	eleventyConfig.addCollection("posts", function (collection) {
-		return collection.getFilteredByGlob("./src/posts/*").sort(function (a, b) {
+		return collection.getFilteredByGlob("./posts/*.md").sort(function (a, b) {
```
Comment (mxmason) [inline .eleventy.js:63]: 5️⃣ these tweks were necessary to prevent a `PrematureUsage` error at build time

**Review by mxmason — COMMENTED** [review]
I left some line-by-line explanations for the changes I had to make to get us to 1.0!

File: .eleventy.js (hunk 1 above)
Comment (ericwbailey) [reply .eleventy.js:6]: Is this the slugify that includes these special characters? I remember that being the reason we had to initially include it.

File: package.json [hunk 6]
```diff
@@ -23,12 +23,12 @@
...
 	},
 	"scripts": {
-		"start": "eleventy --watch & gulp watch",
+		"start": "npm run eleventy-watch & npm run gulp-watch",
 		"a11yproject": "npm start",
 		"publish": "eleventy & gulp",
 		"clean": "rm -rf ./dist",
 		"eleventy-build": "eleventy",
-		"eleventy-watch": "eleventy --watch",
+		"eleventy-watch": "eleventy --incremental --watch",
```
Comment (ericwbailey) [inline package.json:31]: Oooh, good call with `--incremental`!

**Review by ericwbailey — APPROVED** [review]
This is fantastic, thank you! 

@SaptakS, mind weighing in as well if you have the capacity?

File: src/_includes/checklist__task.njk [hunk 7]
```diff
@@ -1,4 +1,4 @@
-{% set slugifiedTitle = task.title | slugify %}
+{% set slugifiedTitle = task.title | striptags(true) | slugify %}
```
Comment (mxmason) [inline src/_includes/checklist__task.njk:1]: @ericwbailey @SaptakS 🔍 

nunjucks' built-in `striptags` handles the case I wrote the extra `slugify` filter for: the `code` tags in the checklist task names

**Comment by SaptakS** [comment]
I couldn't find much info on whether just changing to eleventy slugify would use a similar configuration like we had. Apart from that, rest looks good to me.

**Pull request merged**